mock/service/node.go

```diff
@@ -64,7 +140,7 @@ func (s *service) NodePublishVolume(
 	}
 
 	// Publish the volume.
-	v.Attributes[nodeMntPathKey] = device
+	v.Attributes[nodeMntPathKey] = req.GetStagingTargetPath()
```

Comment:
Changed this from device to the staging target path. I think staging path is bind mounted to the target path. Correct me if I'm wrong about that.

Reply:
I'm not sure. I'll ask.

Reply:
this should only be the staging target path if it is defined - if not, it would be device.

Reply:
Right. Thanks!